h/search/client.py

```diff
@@ -42,28 +38,35 @@ def conn(self):
         return self._conn
 
 
-def get_client(settings):
-    """Return a client for the Elasticsearch index."""
-    host = settings['es.host']
-    index = settings['es.index']
-    kwargs = {}
-    kwargs['max_retries'] = settings.get('es.client.max_retries', 3)
-    kwargs['retry_on_timeout'] = settings.get('es.client.retry_on_timeout', False)
-    kwargs['timeout'] = settings.get('es.client.timeout', 10)
+def _get_settings(settings, elasticsearch=elasticsearch):
+    kwargs = {'verify_certs': True,
+              'max_retries': settings.get('es.client.max_retries', 3),
+              'retry_on_timeout': settings.get('es.client.retry_on_timeout', False),
+              'timeout': settings.get('es.client.timeout', 10)}
 
     if 'es.client_poolsize' in settings:
         kwargs['maxsize'] = settings['es.client_poolsize']
 
     have_aws_creds = ('es.aws.access_key_id' in settings and
                       'es.aws.region' in settings and
                       'es.aws.secret_access_key' in settings)
-
     if have_aws_creds:
         auth = AWS4Auth(settings['es.aws.access_key_id'],
                         settings['es.aws.secret_access_key'],
                         settings['es.aws.region'],
                         'es')
         kwargs['http_auth'] = auth
-        kwargs['connection_class'] = RequestsHttpConnection
+        kwargs['connection_class'] = elasticsearch.RequestsHttpConnection
+    return kwargs
+
+
+def get_client(settings):
```

Comment:
Here we are returning the client connection as opposed to our custom Client wrapper obj.